packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -1133,6 +1139,11 @@ class WebChromeClient extends JavaObject {
   final void Function(WebChromeClient instance, ConsoleMessage message)?
       onConsoleMessage;
 
+  final Future<void> Function(JavaScriptDialogData data)? onJsAlert;
```

Comment:
I may have missed these before, but this callback method should have the same method signature as the Java method: https://developer.android.com/reference/android/webkit/WebChromeClient#onJsAlert(android.webkit.WebView,%20java.lang.String,%20java.lang.String,%20android.webkit.JsResult).  As in `(WebView, String,  String)`. 

The callback methods below should also follow the method signature of their Java methods as well.